src/index.js

```diff
@@ -1,21 +1,76 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-plusplus */
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
 import './style.css';
+import * as allFun from './modules/functions.js';
 
 const Tasks = document.querySelector('.tasks');
+const Input = document.querySelector('form input');
+const AddBtn = document.querySelector('form button');
+const Span = document.querySelector('span');
 
-const tasks = [
-  { index: 1, completed: false, description: 'Wash dishes' },
-  { index: 2, completed: false, description: 'Best wishes' },
-];
+// eslint-disable-next-line import/prefer-default-export, prefer-const
+export let tasks = JSON.parse(localStorage.getItem('tasks')) === null ? [] : JSON.parse(localStorage.getItem('tasks'));
+
```

Comment:
Good job with CRUD implementation so far, however, let's ensure to improve upon the add, edit, delete aspects. When an item is added, it stores correctly in the local storage, but does not display in the UI. A `TypeError` is thrown in this manner, as can be seen from the browser console. I'm unable to perform other CRUD actions such as _edit_ and _delete_ due to this error, and similar, not finding the _delete_ and edit methods you've implemented in this regard. Kindly ensure that:
- [ ] Added items are displaying correctly on the app UI
- [ ] delete and edit functionality are properly integrated 

**Display errors:**
![image](https://user-images.githubusercontent.com/12745474/221299990-be34f172-09ac-457b-b564-a8e0bf0b36e9.png)

![image](https://user-images.githubusercontent.com/12745474/221300068-39fd19f8-9def-4cdc-8ff1-4b9f07583884.png)